src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -10,29 +10,28 @@ public class FigureSupplier {
     private final Random random = new Random();
     private final ColorSupplier colorSupplier = new ColorSupplier();
 
-    public Figure getRandonFigure() {
+    public Figure getRandomFigure() {
         switch (random.nextInt(FIGURE_BOUND)) {
             case 1:
-                return new Circle(colorSupplier.getRandomColor(), random.nextInt(MAX_BOUND_NUMBER));
+                return new Circle(colorSupplier.getRandomColor(), getRandomNumber());
```

Comment:
Good try, but fix not exactly match to what was mentioned in comment. You need extract the next code
`new Circle(colorSupplier.getRandomColor(), random.nextInt(MAX_BOUND_NUMBER));` 
or `new Circle(colorSupplier.getRandomColor(), getRandomNumber());`
to the new method `getRandomCircle()` and call it here. 
Also please extract methods for others figures in the same way.